console/src/components/UIKit/atoms/Badge/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import StyledBadge, { StyledOwnBadgeProps } from './Badge';
+
+export type AllowedBadges =
+  | ''
+  | 'error'
+  | 'no updates'
+  | 'community'
+  | 'update'
+  | 'event';
+
+interface BadgeProps {
+  type: AllowedBadges;
+}
+
+interface ExtendedBadgeProps extends BadgeProps, StyledOwnBadgeProps {}
+
+export const Badge: React.FC<ExtendedBadgeProps> = ({
+  type = '',
+  ...props
+}) => {
+  switch (type) {
+    case 'event':
+      return (
+        <StyledBadge {...props} bg="#001934" color="white">
```

Comment:
I would expect these colours to be defined somewhere in the design system

Reply:
Yup, once the functionality is done, we can focus on this aspect.